source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -175,12 +175,13 @@ def create_pb_request(requested_type, request, dep_mode, arr_mode, direct_path_t
     sn_params.max_bike_duration_to_pt = request["max_bike_duration_to_pt"]
     sn_params.max_bss_duration_to_pt = request["max_bss_duration_to_pt"]
     sn_params.max_car_duration_to_pt = request["max_car_duration_to_pt"]
-    sn_params.max_car_no_park_duration_to_pt = request["max_car_no_park_duration_to_pt"]
+    # So far, ridesharing is the only mode using 'car_no_park' in new_default
+    sn_params.max_car_no_park_duration_to_pt = request["max_ridesharing_duration_to_pt"]
```

Comment:
Whooo, this is SF for me... **Ridesharing** params are used to fill **car_no_park**